src/decorator/options/IndexOptions.ts

```diff
@@ -14,6 +14,16 @@ export interface IndexOptions {
      */
     spatial?: boolean;
 
+    /**
+     * The PG_TEXT_SEARCH_INDEX modifier creates an index to improve the performance of full text searches.
+     * Works only in PostgreSQL.
+     * https://www.postgresql.org/docs/10/textsearch-indexes.html
+     */
+    pgTextSearchIndex?: {
```

Comment:
a couple things that don't conform to the existing standard here:
- prefixing by database type is not something that's used elsewhere, event if it only works in postgres and mysql
- suffixing with Index isn't used
- options are typically just a string not a variable (I think this is fine though)

Reply:
Updated. Removed `pgTextSearchIndex` and used the `fulltext` option instead.